mlos_bench/mlos_bench/tests/optimizers/grid_search_optimizer_test.py

```diff
@@ -239,7 +239,7 @@ def test_grid_search_async_order(grid_search_opt: GridSearchOptimizer) -> None:
     assert all(suggestion.get_param_values() not in suggested_shuffled for suggestion in suggested)
 
     grid_search_opt.bulk_register([suggestion.get_param_values() for suggestion in suggested],
-                                  [score] * len(suggested),
+                                  [{"score": score}] * len(suggested),
```

Comment:
Add a test with multiple `{"metric_1": score_1, "metric_2": score_2}`?

Reply:
It requires a bit more plumbing, so I've implemented it in the next PR #726